### ISSUE TITLE
up-target on form with method=get causes query parameters to repeat

### ISSUE THREAD

**Issue by nshafer** [unpoly/unpoly#76]
When using `up-target` on a form with method=get and no action, such as:

```
<form action="" method="get" up-target=".container">
  <input type="search" name="q"/>
</form>
```

the form works, and the get happens, but every time it will append the query parameters to the URL instead of replacing them with the values set in the form.

I have created a sample of this erroneous behavior at https://codepen.io/nshafer/pen/bJbgPd

Enter anything in the first box a few times, which does a GET without an `up-target` set, and it will behave properly.

Enter anything in the second box, then enter something again, and you'll see the query params keep getting appended to, instead of replaced.

Thanks!
Nate

**Comment by triskweline** [comment]
I've never worked with a form that has no `[action]`. Is the standard behavior documented somewhere? Then Unpoly should match that.

**Comment by nshafer** [comment]
The standard action is to submit the form to the current URL of the form. So if I'm at "/user/account/" then a `<form action="" method="post">` form would submit to "/user/account/". This allows you to write forms that don't need to know about their own URL, such as reusable form snippets included in multiple pages. It's [part of the HTML standard](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#form-submission-algorithm) (step 12) though there seems to be some [debate](https://stackoverflow.com/questions/1131781/is-it-a-good-practice-to-use-an-empty-url-for-a-html-forms-action-attribute-a) about whether it should be blank or not included. I've always had it work just fine by including it and making it blank (`action=""`), whereas I feel that not including `action` seems to cause some issues here and there.

I feel like I've been doing this for over a decade and never had a problem with any browser, and I've seen it used in a lot of web software here and there, but, weirdly, there's a note about [firefox supporting empty action attributes](https://www.fxsitecompat.com/en-CA/docs/2017/html-form-s-action-property-now-returns-document-url-if-action-attribute-is-empty-or-missing/) just a couple years ago, but I swear it worked fine before that. They note that they fixed the bug to be in compliance with the HTML spec.

BTW, I've been using unpoly on a new client Django project for a couple weeks, and so far it's been awesome, really enjoying it. Other than these edge cases, it seems to be very stable and fast. I'm excited to continue using it for future projects as well. You are absolutely right in that full SPAs with React or Vue are absolutely unnecessary for most web projects, but this makes them feel like it without all the extra complications and technical debt. Kudos. The one thing that's driving me crazy... why is it called, "Unpoly?"

**Comment by triskweline** [comment]
Fixed in 0.60.2.

**Comment by nshafer** [comment]
Thank you!
